tests/test_on_data/test_recording_interfaces.py

```diff
@@ -84,6 +85,17 @@ class TestCEDRecordingInterface(RecordingExtractorInterfaceTestMixin, TestCase):
     save_directory = OUTPUT_PATH
 
 
+class TestCellExplorerRecordingInterface(RecordingExtractorInterfaceTestMixin, TestCase):
+    data_interface_cls = CellExplorerRecordingInterface
+    interface_kwargs = [
+        dict(folder_path=str(DATA_PATH / "cellexplorer" / "dataset_4" / "Peter_MS22_180629_110319_concat_stubbed")),
+        dict(
+            folder_path=str(DATA_PATH / "cellexplorer" / "dataset_4" / "Peter_MS22_180629_110319_concat_stubbed_hdf5")
+        ),
+    ]
+    save_directory = OUTPUT_PATH
```

Comment:
Can you (i) break these into two separate testing classes so that (ii) you can add separate `check_metadata` tests to showcase all the metadata info they are automatically collecting?

Reply:
It is the same data in different formats. The metadata test if any should be the same.

Reply:
Added the test for channel metadata extraction